wiring/inc/spark_wiring_ble.h

```diff
@@ -147,6 +147,33 @@ enum class BleTxRxType : uint8_t {
     NACK = 2
 };
 
+enum class BlePairingIoCaps : uint8_t {
+    NONE = BLE_IO_CAPS_NONE,
+    DISPLAY_ONLY = BLE_IO_CAPS_DISPLAY_ONLY,
+    DISPLAY_YESNO = BLE_IO_CAPS_DISPLAY_YESNO,
+    KEYBOARD_ONLY = BLE_IO_CAPS_KEYBOARD_ONLY,
+    KEYBOARD_DISPLAY = BLE_IO_CAPS_KEYBOARD_DISPLAY
+};
+
+enum class BlePairingEventType : uint8_t {
+    REQUEST_RECEIVED = BLE_EVT_PAIRING_REQUEST_RECEIVED,
+    PASSKEY_DISPLAY = BLE_EVT_PAIRING_PASSKEY_DISPLAY,
+    PASSKEY_INPUT = BLE_EVT_PAIRING_PASSKEY_INPUT,
+    STATUS_UPDATED = BLE_EVT_PAIRING_STATUS_UPDATED
+};
+
+union BlePairingEventPlayload {
```

Comment:
I think this might be a typo. Should it be renamed to `BlePairingEventPayload`?

Reply:
Good catch! Changed to `BlePairingEventPayload`.